common/unit_test/Test_Extents.cpp

```diff
@@ -6,9 +6,10 @@
 #include <Kokkos_Random.hpp>
 #include <vector>
 #include "KokkosFFT_Extents.hpp"
-#include "Test_Types.hpp"
 #include "Test_Utils.hpp"
 
+namespace {
+using execution_space = Kokkos::DefaultExecutionSpace;
```

Comment:
Did you make sure it is allowed to define google tests inside anonymous namespaces ?

Reply:
If I remembered, you encountered an issue with DDC. 
Do you remember what was that?

Reply:
In DDC, we usually define a `struct` with the same name in different source files. We then define them in anonymous namespaces to avoid conflicts in the test executable.

Reply:
Do you mean defining a fixture with the same name in different source files?
Then, gtest will raise errors about the symbol collision, if we define tests inside anonymous namespace.

Reply:
Sure
Maybe it is safer to use normal namespaces?
I am using anonymous namespace in order to avoid the overlapping of fixture names in different files.

Reply:
Based on the discussion yesterday, I have moved `TEST_* marcos` outside an anonymous namespace.